salt/dex/dex.yaml

```diff
@@ -0,0 +1,136 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dex
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+---
+# Map the LDAP Administrators role to the Kubernetes system:masters group
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: system:dex
+subjects:
+- kind: ServiceAccount
+  name: dex
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dex
+  namespace: kube-system
+data:
+  config.yaml: |
+    issuer: "https://{{ pillar['api']['server']['external_fqdn'] }}:{{ pillar['dex']['node_port'] }}"
+    storage:
+      type: kubernetes
+      config:
+        inCluster: true
+    web:
+      https: 0.0.0.0:5556
+      tlsCert: /etc/dex/tls/dex.crt
+      tlsKey: /etc/dex/tls/dex.key
+    connectors:
+    - type: ldap
+      id: ldap
+      name: LDAP
+      config:
+        host: "{{ pillar['dashboard'] }}:389"
+        startTLS: true
+        bindDN: cn=admin,{{ pillar['ldap_internal_infra_domain'] }}
+        bindPW: admin
+        rootCA: {{ pillar['ssl']['ca_file'] }}
+        userSearch:
+          baseDN: ou=People,{{ pillar['ldap_internal_infra_domain'] }}
+          filter: "(objectClass=inetOrgPerson)"
+          username: mail
+          idAttr: DN
+          emailAttr: mail
+          nameAttr: cn          
+        groupSearch:
+          baseDN: ou=Groups,{{ pillar['ldap_internal_infra_domain'] }}
+          filter: "(objectClass=groupOfUniqueNames)"
+
+          userAttr: DN
+          groupAttr: uniqueMember
+
+          nameAttr: cn       
+
+    oauth2:
+      skipApprovalScreen: true
+
+    staticClients:
+    - id: caasp-cli
+      redirectURIs:
+      - 'http://127.0.0.1'
```

Comment:
Shouldn't this be set to the special out of browser URI? `urn:ietf:wg:oauth:2.0:oob`